static/app/views/replays/detail/ai/ai.tsx

```diff
@@ -56,9 +56,19 @@ export default function Ai() {
     );
   }
 
+  if (isPending || isRefetching) {
+    return (
+      <Wrapper data-test-id="replay-details-ai-summary-tab">
+        <LoadingContainer>
+          <LoadingIndicator />
+        </LoadingContainer>
+      </Wrapper>
+    );
+  }
+
   // If our `replay-ai-summaries` ff is enabled and the org has gen AI ff enabled,
   // but the org hasn't acknowledged the gen AI features, then show CTA.
-  if (!setupAcknowledgement.orgHasAcknowledged) {
+  if (!isPending && !isRefetching && !setupAcknowledgement.orgHasAcknowledged) {
```

Comment:
is `!isPending && !isRefetching` needed here? we should return earlier if either of those are still set.